Dalal_Saeed/week_04/day_04/app.rb

```diff
@@ -0,0 +1,79 @@
+require "bundler/setup" # require all the gems we'll be using for this app from the Gemfile. Obviates the need for `bundle exec`
+require "active_record" # the ORM
+
+require_relative "db/connection" # require the db connection file that connects us to PSQL, prior to loading models
+require_relative "models/artist" # require the Artist class definition that we defined in the models/artist.rb file
+require_relative "models/song" # require the Song class definition that we defined in the models/song.rb file
+
+#Artist.create({name: "Jack", age: 89})
+# Part 0
+# - install the necessary gems!
+# - Create a SQLite database.  Run the schema and seed the data.
+
+# Part 1.1
+# There are a bunch of challenges to find different models below.
+# write the Active Record code to complete the challenge below each.
+
+# Find all artists
+all_artist = Artist.all
+all_artist
+
+# Find just one artist by id
+p Artist.find(3)
+# Find Taylor Swift (or your other fav artist) by name.
+p Artist.find_by name: "Taylor Swift"
+# Find all artists from the USA
+p Artist.find_by nationality: "American" #wrong
+# Find all artists NOT from the USA
+all_artist2 = Artist.not.where("nationality =American")
+# Create a new artist for your favorite artist
+hana = Artist.create({name: "Hana", photo_url: "photo"})
+# Change at least 2 of their attributes
```

Comment:
Good start @dalalsa, try to get these tasks done as it'll be very handy throughout the course!